dashboard/grid_builder.py

```diff
@@ -88,8 +96,32 @@ def has_issues(self, p: package.Package) -> bool:
                 if not package_2_self_conflict and \
                                 result['status'] != 'SUCCESS':
                     return True
+
+            # Whether the package has deprecated dependencies or not
+            if self.deprecated_deps[p.install_name][1]:
+                return True
+
         return False
 
+    def get_deprecated_deps(self):
+        """
+        Returns True if there are deprecated dependencies for a
```

Comment:
Maybe add the type annotation? This doesn't return a bool.

Reply:
Done.